src/main/java/com/thirdcc/webapp/config/ApplicationProperties.java

```diff
@@ -20,4 +22,61 @@ public boolean isShouldInitFirebase() {
     public void setShouldInitFirebase(boolean shouldInitFirebase) {
         this.shouldInitFirebase = shouldInitFirebase;
     }
+
+    public AccessToken getAccessToken() {
+        return accessToken;
+    }
+
+    public RefreshToken getRefreshToken() {
+        return refreshToken;
+    }
+
+    public static class AccessToken {
```

Comment:
is it good to put AccessToken and RefreshToken in Application Properties?
what about in security/jwt directory?

Reply:
My decision to put `AccessToken` & `RefreshToken` as inner static class in `ApplicationProperties` are because
1. sometimes it seems most sensible to sort a class into the namespace of another class, especially when it won't be used in any other context
2. The `JhipsterProperties` at `package io.github.jhipster.config;` does the same thing
3. for convenience

Reply:
ok got it
thanks 👍